nxos_bgp.tf

```diff
@@ -187,3 +187,38 @@ resource "nxos_bgp_peer_address_family" "bgpPeerAf" {
   send_community_extended = each.value.send_community_extended
   send_community_standard = each.value.send_community_standard
 }
+
+locals {
+  routing_bgp_vrfs_neighbors_address_families_route_control = flatten([
+    for device in local.devices : [
+      for vrf in try(local.device_config[device.name].routing.bgp.vrfs, []) : [
+        for nei in try(vrf.neighbors, []) : [
+          for af in try(nei.address_families, []) : (
+            try(af.route_map_name, null) != null ? [
+              {
+                key            = format("%s/%s/%s/%s", device.name, vrf.vrf, nei.ip, local.address_family_names_map[af.address_family])
+                device         = device.name
+                vrf            = vrf.vrf
+                neighbor_key   = format("%s/%s/%s", device.name, vrf.vrf, nei.ip)
+                address_family = local.address_family_names_map[af.address_family]
+                route_map_name = af.route_map_name
+                direction      = try(af.route_map_direction, local.defaults.nxos.configuration.routing.bgp.vrfs.neighbors.address_families.route_map.route_map_direction, "in")
+              }
+            ] : []
+          )
+        ]
+      ]
+    ]
+  ])
+}
+
+resource "nxos_bgp_peer_address_family_route_control" "bgp_peer_af_route_control" {
```

Comment:
By convention, the resource name should be the type without the `nxos_` prefix -> `bgp_peer_address_family_route_control `

Reply:
This one I am confused about. 
`nxos_bgp_peer_address_family_route_control` is the resource from the nxos provider while `bgp_peer_af_route_control` is the given name for that resource inside the module.
If I were to remove `nxos` from `nxos_bgp_peer_address_family_route_control` it would result in not being found in the nxos provider.

Reply:
What I meant is this:

```
resource "nxos_bgp_peer_address_family_route_control" "bgp_peer_address_family_route_control" {
```